src/sdk/python/rtdip_sdk/queries/time_series/interpolation_at_time.py

```diff
@@ -47,7 +47,7 @@ def get(connection: object, parameters_dict: dict) -> pd.DataFrame:
         raise ValueError("timestamps must be a list")
 
     try:
-        query = _query_builder(parameters_dict, metadata=False, interpolation_at_time=True)
+        query = _query_builder(parameters_dict, "interpolation_at_time", interpolation_at_time=True)
```

Comment:
why would you still need the `interpolation_at_time` boolean flag if the query_type is set with "interpolation_at_time"?

Reply:
I was doing each function one at a time, so I haven't started interpolation at time yet, i just added that query_type parameter so that it didn't fail if someone tries to use it in dev.

Reply:
ok cool :)